### DISCUSSION TITLE
Min and max length rules

### DISCUSSION THREAD

**Discussion by yuges92** [awcodes/filament-tiptap-editor#401] [Ideas]
Can a minimum or maximum length rule be added to the editor?
It does not respect the validation rules when setting the maximum rule.

eg: 
```php
 TiptapEditor::make('text')
             ->label('Text')
             ->columnSpanFull()
             ->rules([
                'max:500'
             ]),
```

**Comment by awcodes** [comment]
This is kinda of an iffy subject. Since the data can be stored as html or json, the amount of actual text can vary differently and drastically. For example, a 4 word sentence could have 100 extra characters of html. So, how can a min a max validation be reasonably applied to that?

**Reply by yuges92** [reply]
Thanks for the clarification. Agreed, it makes sense not to have the min and max rules
